ush/forecast_postdet.sh

```diff
@@ -675,13 +675,19 @@ GOCART_rc() {
   # set input directory containing GOCART input data and configuration files
   # this variable is platform-dependent and should be set via a YAML file
 
+  local vdate
   # link directory containing GOCART input dataset, if provided
   if [[ -n "${AERO_INPUTS_DIR}" ]]; then
     ${NLN} "${AERO_INPUTS_DIR}" "${DATA}/ExtData"
     status=$?
     [[ ${status} -ne 0 ]] && exit "${status}"
   fi
 
+  # Link blending emissions if AERO_EMIS_FIRE == blending
+  if [[ "${AERO_EMIS_FIRE}" == 'blending' ]]; then 
```

Comment:
I think this is only relevant for `RUN=gefs`, so we should add that conditionality. Atleast till such time that `RUN=gfs` also runs w/ aerosols
```suggestion
  if [[ "${AERO_EMIS_FIRE}" == "blending" && "${RUN}" == "gefs" ]]; then 
```

Reply:
GDAS_fcst will run with aerosols

Reply:
Thanks @CoryMartin-NOAA 
Will `GDAS_fcst` use blended emissions?  Presently, this task is currently only activated for GEFS (and SFS) -- at least that is what was communicated.

Reply:
oh, I don't know, @bbakernoaa any thoughts?

Reply:
If the GDAS forecast is less than a day or two we definitely don't need this at all.  The current setup is to only add the blending after 5 days of assumed persistence.

Reply:
9 hours, max, so we don't need it for GDAS then. Thanks